games-engines/fheroes2/fheroes2-1.0.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop
+
+DESCRIPTION="Recreation of HoMM2 game engine"
+HOMEPAGE="https://ihhub.github.io/fheroes2/"
+SRC_URI="https://github.com/ihhub/fheroes2/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="tools"
+
+COMMON_DEPEND="
+	media-libs/libpng:=
+	media-libs/libsdl2[video]
+	media-libs/sdl2-image
+	media-libs/sdl2-mixer
+	sys-libs/zlib
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	app-arch/libarchive
+	virtual/libintl
+"
```

Comment:
I think you need to add `dev-lang/python` here for `extract_homm2_resources.sh` ? Technically, we should only support specific versions, but that might be overkill when it's only for a few short lines.

Reply:
Done